### PR TITLE
frontend: update pocket T&C and info box content, remove unused translation keys

### PR DESCRIPTION
**Pull request by shonsirsha**
moved & update content of pocket T&C. Removed the old / out-of-date (thus unused) pocket T&C translation keys.

T&C: 

tnc_

Info box: 

ibox

### CODE REVIEW

File: frontends/web/src/locales/en/app.json [hunk 1]
```diff
@@ -428,23 +428,23 @@
...
-              "uk": "UK Faster Payments (GBP transactions in the UK only)"
+              "uk": "UK Faster Payments (GBP)"
             },
             "bankTransferReccuring": "How to set up recurring purchases with a standing order?",
             "title": "Payment methods"
           },
           "supportedCurrencies": "Supports European currencies: EUR, GBP, and CHF.",
           "verification": {
-            "info": "Only requires identity verification above daily and annual thresholds.",
+            "info": "No identitiy verification for amounts below threshold (~950 EUR/day)",
```
Comment (shonsirsha) [inline frontends/web/src/locales/en/app.json:447]: These 4 keys need to be updated manually on locize once `staging-pocketsell` is merged to `master`.

File: frontends/web/src/locales/en/app.json [hunk 2]
```diff
@@ -719,6 +693,38 @@
...
+    "pocket": {
+      "terms": {
+        "dataprotection": {
+          "link": "Pocket privacy policy",
+          "p1": "The BitBoxApp does not learn or collect any personal data. Pocket collects some personal data as explained in their Privacy Policy.",
+          "title": "Data Protection"
+        },
+        "fees": {
+          "note": "Pocket's exchange rates may differ from the BitBoxApp rates.",
+          "p1": "Buying bitcoin: Use SEPA bank transfer to send EUR or CHF and get bitcoin delivered directly to your BitBox after Pocket receives the transfer, usually the same day.",
```
Comment (Beerosagos) [inline frontends/web/src/locales/en/app.json:706]: I think "Buying/Selling bitcoin" should be bold

File: frontends/web/src/locales/en/app.json [hunk 3]
```diff
@@ -719,6 +693,38 @@
...
+    },
+    "pocket": {
+      "terms": {
+        "dataprotection": {
+          "link": "Pocket privacy policy",
+          "p1": "The BitBoxApp does not learn or collect any personal data. Pocket collects some personal data as explained in their Privacy Policy.",
+          "title": "Data Protection"
+        },
+        "fees": {
+          "note": "Pocket's exchange rates may differ from the BitBoxApp rates.",
```
Comment (Beerosagos) [inline frontends/web/src/locales/en/app.json:705]: Missing 'Note:' at the beginning in the string

File: frontends/web/src/locales/en/app.json (hunk 1 above)
Comment (Beerosagos) [reply frontends/web/src/locales/en/app.json:447]: Do we want to keep this commit anyway? Also, I think it still misses the section to say that to sell you need to have bought with pocket before, as @jadzeidan suggested. Maybe he can draft a wording for that?

**Review by Beerosagos — COMMENTED** [review]
Thanks a lot :pray: 

I left a couple of minor comments around

File: frontends/web/src/locales/en/app.json (hunk 1 above)
Comment (shonsirsha) [reply frontends/web/src/locales/en/app.json:447]: Will wait for the draft 👍 thanks.

**Comment by shonsirsha** [comment]
Thanks for the review, merged the 2 commits into one. Updated the preview, PTAL 🙏 @Beerosagos

**Review by Beerosagos — APPROVED** [review]
LGTM! Thanks :pray:

**Pull request merged**